### PR TITLE
Add autocomplete tag on entry edit

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

File: app/entry/entry_edit/entry_edit_controller.js [hunk 1]
```diff
@@ -13,10 +13,11 @@
  * @param Codekit
  * @param API
  * @param AuthService
- * @oaram DolphinService
+ * @param DolphinService
+ * @param $q
  */
 function EntryEditController($scope, $rootScope, $state, $stateParams, $mdToast,
-  Codekit, API, AuthService, DolphinService, $q) {
+                             Codekit, API, AuthService, DolphinService, $q) {
```
Comment (amirm3hdi) [inline app/entry/entry_edit/entry_edit_controller.js]: Revert this indention back to original.

File: app/entry/entry_edit/entry_edit_controller.js [hunk 2]
```diff
@@ -32,18 +33,18 @@ function EntryEditController($scope, $rootScope, $state, $stateParams, $mdToast,
...
-      site: AuthService.getCurrentSite(),
+      site: AuthService.getCurrentSite()
     };
 
-    API.Tags.get({ tag_site: AuthService.getCurrentSite() },
+    API.Tags.get({tag_site: AuthService.getCurrentSite()},
       function (data) {
         for (var i in data.results) {
-          $scope.tags.push({ slug: data.results[i].slug, id: data.results[i].id, name: data.results[i].name, });
+          $scope.tags.push({slug: data.results[i].slug, id: data.results[i].id, name: data.results[i].name});
```
Comment (amirm3hdi) [inline app/entry/entry_edit/entry_edit_controller.js]: Revert.

File: app/entry/entry_edit/entry_edit_controller.js [hunk 3]
```diff
@@ -32,18 +33,18 @@ function EntryEditController($scope, $rootScope, $state, $stateParams, $mdToast,
     $scope.statuses = Codekit.entryStatuses;
     $scope.form = {
       id: $stateParams.entryId,
-      site: AuthService.getCurrentSite(),
+      site: AuthService.getCurrentSite()
     };
 
-    API.Tags.get({ tag_site: AuthService.getCurrentSite() },
+    API.Tags.get({tag_site: AuthService.getCurrentSite()},
```
Comment (amirm3hdi) [inline app/entry/entry_edit/entry_edit_controller.js]: Revert.

File: app/entry/entry_edit/entry_edit_controller.js [hunk 4]
```diff
@@ -32,18 +33,18 @@ function EntryEditController($scope, $rootScope, $state, $stateParams, $mdToast,
...
       function (data) {
         for (var i in data.results) {
-          $scope.tags.push({ slug: data.results[i].slug, id: data.results[i].id, name: data.results[i].name, });
+          $scope.tags.push({slug: data.results[i].slug, id: data.results[i].id, name: data.results[i].name});
         }
       }
     );
 
-    API.Entry.get({ entry_id: $scope.form.id },
+    API.Entry.get({entry_id: $scope.form.id},
```
Comment (amirm3hdi) [inline app/entry/entry_edit/entry_edit_controller.js]: Revert.

File: app/entry/entry_edit/entry_edit_controller.js [hunk 5]
```diff
@@ -97,18 +100,18 @@ function EntryEditController($scope, $rootScope, $state, $stateParams, $mdToast,
...
    *
    * @method remove
    * @desc remove entry via api call
-   * 
+   *
    * @param id {string} UUID of entry
    */
   $scope.remove = function (id) {
-    API.Entry.delete({ entry_id: id },
+    API.Entry.delete({entry_id: id},
```
Comment (amirm3hdi) [inline app/entry/entry_edit/entry_edit_controller.js]: Revert.

**Comment by amirm3hdi** [comment]
image

Fixed them all and added some styles.

**Comment by Alir3z4** [comment]
This is really coming nice.
Are those the Tag's slug or their name?

On Sun, Nov 20, 2016 at 9:14 PM, Amir Mehdi Savand <notifications@github.com

**Pull request merged**